commands/miscellaneous/getStudentProfile.js

```diff
@@ -6,61 +6,60 @@ const ALL_GUILD_IDS = [BATId, BALId, BAGId, BADId, guildId];
 
 module.exports = {
   data: new SlashCommandBuilder()
-    .setName('getstudentprofile')
-    .setDescription('Gets info of a students across BA servers')
-    .setDefaultPermission(false)
-    .addUserOption(option =>
-      option.setName('user')
-        .setDescription('The user you want to get their profile.')
-        .setRequired(true)),
+      .setName('getstudentprofile')
+      .setDescription('Gets info of a students across BA servers')
+      .setDefaultPermission(false)
+      .addUserOption(option =>
+          option.setName('user')
+              .setDescription('The user you want to get their profile.')
+              .setRequired(true)),
   async execute(interaction) {
     const options = interaction.options;
     const user = options.getUser('user');
     const fields = [];
 
-    const formatStudentInfo = (guildName, value) => {
+    const formatField = (displayName, value) => {
       const info = {
-        name: `Roles at ${guildName}`,
+        name: `${displayName}: `,
         value: `${value}`,
-        inline: true
+        inline: false
       };
       return info;
     };
 
     await interaction.deferReply();
+    let userInGuild = null;
     ALL_GUILD_IDS.forEach(guildId => {
       const guild = interaction.client.guilds.cache.get(guildId);
       if (guildId && !guild) return;
-      const userInGuild = guild.members.cache.find(u => u.id === user.id);
+      userInGuild = guild.members.cache.find(u => u.id === user.id);
       if (!userInGuild) return;
       let roles = userInGuild.roles.cache.filter(role => role.name != '@everyone');
       if (interaction.guild == guildId) {
         roles = roles.map(role => `<@&${role.id}>`).join('\n');
       } else {
         roles = roles.map(role => role.name).join('\n');
       }
-      const joinedDate = new Date(userInGuild.joinedAt).toLocaleString('ko-KR', { timeZone: 'UTC' });
```

Comment:
this looks like a lot of changes but i just cleaned up the code a lil bit so that we're adding fields the same way

Reply:
i think we want to keep the joinedDate inside the loop so we get the date they joined per server? and it looks like we may have been setting the roles and joined date in one result like this so it was always formatted together and not split in a weird way by discord wrapping on the embed?